vendor/assets/javascripts/announcements.js

```diff
@@ -0,0 +1,29 @@
+function hideAnnouncement(announcement_key) {
```

Comment:
The docs implied that the js file was required. After testing this, it seems that the js file and require are unnecessary.